coremark/idf_component.yml

```diff
@@ -0,0 +1,8 @@
+version: "1.1.0"
```

Comment:
I was unable to find upstream version, only see 1.01 tag there. Should we start with same tag here?

Reply:
I interpreted 1.01 = 1.1 = 1.1.0, what would be your suggestion?

Reply:
Hmm..okay. I guess we can continue with `1.1.0` here.